src/controllers/productController.ts

```diff
@@ -56,7 +42,18 @@ async function retrieveProduct(req: Request, res: Response, next: NextFunction)
 
 async function indexProduct(req: Request, res: Response, next: NextFunction) {
   try {
-    const products = await Product.findAll();
+    const products = await Product.findAll({
+      include: [
+        {
+          association: Product.associations.Stall,
+          include: [
+            {
+              association: Stall.associations.HawkerCentre,
+            },
+          ],
+        },
+      ],
+    });
     req.body = products;
```

Comment:
Can just end the request by doing`res.status(200).json(products);`, no need this line and the next line.

Reply:
Removed